### ISSUE TITLE
Translation for other languages

### ISSUE THREAD

**Issue by lex111** [gitpoint/git-point#439]
Hello everyone!

We need new translations, **the instruction to add is [here](https://github.com/gitpoint/git-point/wiki/I18N#adding-a-new-language)**.

At the moment **existing** translations:
- English
- French
- Dutch
- Turkish
- Brazilian Portuguese
- Russian
- Galician (thanks to @siderio2!)
- Spanish
- Portuguese (thanks to @Jpfonseca!)
- Portuguese Brazilian
- German (thanks to @tautf!)
- Polish (thanks to @peterblazejewicz!)
- Esperanto (thanks to @kodisto!)
- Ukrainian
- Chinese Traditional
- Chinese Simplified
- Basque
- Swedish (thanks to @SpaceLenore)
- Tagalog (thanks to @geeofree)
- Serbian (thank to @cobisimo)

We are looking forward to the _new_ translations, this will be a good start for Hacktoberfest, so welcome!

**P.S. Only native speakers, please.**

**Comment by khvilaboa** [comment]
Spanish coming soon... :)

**Comment by lex111** [comment]
@khvilaboa it is very good! However, we already have PR #282 for translating into Spanish, but it is not finished, and you can take it as a basis and create your own PR with already ready translation, I hope, @alejandronanez  will not mind, right? @alejandronanez  will then check the accuracy of the translation in that case.

**Comment by stephenpfaff** [comment]
So you need **_any_** language which does not already exist then or specific ones?

**Comment by lex111** [comment]
@stephenpfaff we will be glad to **any** language for which there is no translation yet.

**Comment by tminussi** [comment]
I would be happy to help with pr-br

**Comment by housseindjirdeh** [comment]
Thank you all so much, _any language_ not present will be AMAZING

**Comment by Rishabh04-02** [comment]
I'd like to help adding **de** - German.

**Comment by andrewda** [comment]
@Rishabh04-02 Awesome! Thanks a lot!

**Comment by FlorianTaut** [comment]
@Rishabh04-02 I'm already working on german Translation, but you can make this your Job if you want to. Thank you :+1:

**Comment by Rishabh04-02** [comment]
@FlorianTaut actually I've started to work on this. But so far have done only 10% of the job. So, if your work has progressed more than mine then you can work on it, I'll work on something else. But please let me know, if you're working on this.

**Comment by viettrung9012** [comment]
I can help with translation to Vietnamese

**Comment by phillip055** [comment]
Thai!

**Comment by peterblazejewicz** [comment]
Hi, I'll add Polish (pl-Pl). Thanks!

**Comment by lex111** [comment]
@peterblazejewicz what for an error occurs? We will help you to solve this, you can also write to our chat on [Gitter](http://gitter.im/git-point/Lobby)

**Comment by Eliottiosdevs** [comment]
#451 updated Dutch phrases

**Comment by idealadarsh** [comment]
Hello, I'd like to add Hindi language.

**Comment by housseindjirdeh** [comment]
![gif](https://media.giphy.com/media/RrVzUOXldFe8M/giphy.gif)

You're all awesome :tada: :tada: THE NEXT RELEASE IS GOING TO :fire: with all these languages

**Comment by housseindjirdeh** [comment]
Yep just as @lex111 mentioned, we'll be more than happy to help any issues you're having @peterblazejewicz!

**Comment by tminussi** [comment]
Awesome!

**Comment by alejandronanez** [comment]
Yesssssss

**Comment by peterblazejewicz** [comment]
@lex111 @housseindjirdeh It appears to be iOS cache problem in sim. The local build works after I cleaned it up.
So I'll do 'pl-Pl' (pl.js). I'll ask question on the gitter to not add more noise here.
Thanks!

**Comment by BeaData** [comment]
I'll be doing the Italian (it-It) translation :)

**Comment by FlorianTaut** [comment]
@BeaData Thank you very much! [Here](https://github.com/gitpoint/git-point/blob/master/CONTRIBUTING.md#translations) you can find how you can get started!

**Comment by dixi3** [comment]
Hi, i would like to work on the romanian translation

**Comment by FlorianTaut** [comment]
@dixi3 Go for it! We love every new language which is added to gitpoint!

**Comment by Kietzmann** [comment]
I'll add Ukrainian translation

**Comment by hndr91** [comment]
Can I work on id-id ?

**Comment by msarsha** [comment]
Hey!

Will right to left language be properly displayed ?

**Comment by MozyOk** [comment]
Hi, Thank you for a wonderful application!
I'd like to add Japanese translation.

**Comment by machour** [comment]
Hi @msarsha, sorry we missed your comment 😞 

I never was able to gather enough courage to try an Arabic translation for the app, so I'm not sure.. would you like to start a new PR to implement Hebrew? We'll figure how things go from there!

**Comment by machour** [comment]
Hey @MozyOk ! Thank you 🎉 
We would LOVE if you could add Japanese to GitPoint!

**Comment by SpaceLenore** [comment]
I'd like to help with a Swedish translation!  
The Swedish language is interesting, we don't really translate GitHub terms such as repositories, merge, push, commits, etc. It usually becomes more confusing and some times used as a joke i.e. "git knuffa"

**Comment by lex111** [comment]
@SpaceLenore fine! Based on the file from [the English version](https://github.com/gitpoint/git-point/blob/master/src/locale/languages/en.js) and change only the values (the keys remain intact).

**Comment by lex111** [comment]
@geeofree :hand: cool, we expect PR!

**Comment by 94151** [comment]
@housseindjirdeh i want to translate it in hindi. can you guide me more how can i start working on it?

**Comment by ooHAoo** [comment]
Hi, don't see one for Thai (TH) yet so I would like to add one, if possible. Thanks!

**Comment by rudsberg** [comment]
I could help out with some swedish stuff if you guys interested

**Comment by lex111** [comment]
Hey @rudsberg 

There is already a translation in Swedish, but you can check its correctness, for example.

**Comment by EdZava** [comment]
@lex111  pull request some bug fixes of the portuguese translation.

**Comment by ooHAoo** [comment]
Hi, I need some help. Not sure if I should be creating an issue or just post here. Please let me know.
So I have done the translation and everything and now I'm trying to up the app to run on my android but everytime I run this command it gives me an error that BUILD FAILED:

yarn start:android

**Comment by lex111** [comment]
@ooHAoo hello, please give more information? What system do you have, better send a screenshot?

**Comment by olegkorol** [comment]
I could provide a translation for Catalan. Should I?

**Comment by lex111** [comment]
@olegkorol it would be cool! :+1:

**Comment by jatempa** [comment]
I want to translate for spanish

**Comment by alejandronanez** [comment]
Hola @jatempa,

We already have spanish translations, but you can take a look, maybe we missed something!

Please, add me as a reviewer since I’m also a native spanish speaker!

Thanks!

**Comment by lex111** [comment]
@chinesedfan oops, accidentally closed this issue, thanks for opening again!

**Comment by giovannipessiva** [comment]
Seems like Italian is missing, I'll work on it

**Comment by SpaceLenore** [comment]
Hello again, I noticed that there are some new words and phrases added that does not have any Swedish translations yet. I'll submit a PR with them shortly.

**Comment by rakeshlanjewarwork** [comment]
I am working on marathi ( India ).

code = mr

**Comment by raihan71** [comment]
> Hello everyone!
> 
> We need new translations, **the instruction to add is [here](https://github.com/gitpoint/git-point/wiki/I18N#adding-a-new-language)**.
> 
> At the moment **existing** translations:
> 
> * English
> * French
> * Dutch
> * Turkish
> * Brazilian Portuguese
> * Russian
> * Galician (thanks to @siderio2!)
> * Spanish
> * Portuguese (thanks to @Jpfonseca!)
> * Portuguese Brazilian
> * German (thanks to @tautf!)
> * Polish (thanks to @peterblazejewicz!)
> * Esperanto (thanks to @kodisto!)
> * Ukrainian
> * Chinese Traditional
> * Chinese Simplified
> * Basque
> * Swedish (thanks to @SpaceLenore)
> * Tagalog (thanks to @geeofree)
> * Serbian (thank to @cobisimo)
> * Catalan (thank to @olegkorol)
> 
> We are looking forward to the _new_ translations, this will be a good start for Hacktoberfest, so welcome!
> 
> **P.S. Only native speakers, please.**

I can translate it to Indonesia

**Comment by Jpfonseca** [comment]
I will be looking into the Portuguese translation as soon as I can.

Get Outlook for Android<https://aka.ms/ghei36>

________________________________
